build-ci-wheels/action.yml

```diff
@@ -1,57 +1,60 @@
-name: "Build wheels for a C-extension Python library."
-description: "Build wheels for a C-extension Python library."
+name: >
+  Build CI wheels
+
+description: >
+  Build wheels for a Python library. This action is expected to be used within a
+  matrix job. Its goal is to generate wheel files for every combination of
+  operating system, build system, and Python version.
```

Comment:
I would add some kind of disclaimer... this is not always needed. Only if your library has some sort of compiled code. Otherwise it is not applicable and you can live with the default twine wheel.

Reply:
I agree, We can add something like **_Build ci wheels for CPython and PyPy_** or **_Builds manylinux, musllinux, macOS 10.9+, and Windows wheels for CPython and PyPy_** or we can specify it in description.

Reply:
I just improved this description indicating that only libraries shipping with source code that needs to be compiled should make use of this action.